server/lib/payments.js

```diff
@@ -497,13 +497,13 @@ export async function createRefundTransaction(
   }
 
   // Refund Payment Processor Fee
-  await refundPaymentProcessorFee(transaction, user, refundedPaymentProcessorFee, transactionGroup, clearedAt);
+  await refundPaymentProcessorFee(transaction, user, refundedPaymentProcessorFee, transactionGroup, {}, clearedAt);
```

Comment:
Typescript would have helped here

Reply:
Yes, @znarf is already refactoring this file and converting it to TS.